overlord/configstate/configcore/services.go

```diff
@@ -144,7 +145,7 @@ func switchDisableService(serviceName string, disabled bool, opts *fsOnlyContext
 	if opts != nil {
 		sysd = systemd.NewEmulationMode(opts.RootDir)
 	} else {
-		sysd = systemd.NewUnderRoot(dirs.GlobalRootDir, systemd.SystemMode, &sysdLogger{})
```

Comment:
there are more places in  configcore that use NewUnderRoot should we change all of them?

Reply:
Almost certainly yes. As discussed on the channel it will be a followup (having them spread-tested will required some effort and might be tricky).